content/dev/languages/creating_a_language_pack.md

```diff
@@ -0,0 +1,291 @@
+
+- Copy `includes/languages/english.php` to `includes/languages/YOURLANGUAGE.php`
+
+- Copy `includes/languages/english/` and all sub directories to `includes/languages/YOURLANGUAGE/`
+
+- Copy `admin/includes/languages/english.php` to `admin/includes/languages/YOURLANGUAGE.php`
+
+- Copy `admin/includes/languages/english/` and all sub directories to `admin/includes/languages/YOURLANGUAGE/`
+
+If old style image buttons are being used then you will also need to create
+
+- `includes/templates/template_default/buttons/YOURLANGUAGE/` and
+
+- `includes/templates/responsive_classic/buttons/YOURLANGUAGE/` then fill with images of the buttons in your language
+
+### Prior to 1.5.7
+
+Prior to version 1.5.7  the `emails` folder contained some English language sections in the templates. 
+
+- `emails` modify the email template files contents from English to your language (these files will overwrite the original distribution files).
+
+### Admin Extras
+
+The Admin pages on zen cart hold a lot of information in the configuration table on the database. To do a complete conversion it will be necessary to convert the description fields to your chosen language. This will require a SQL patch.
+
+### Locations of language files
+
+These are the locations where you will find language files that may need translating.
+
+- emails **NOTE** Only required for v1.5.6 and earlier
+
+    - email_common.css
+
+    - email_template_checkout.htm
+
+    - more html files
+
+- admin
+
+    - includes
+
+        - languages
+
+            - english.php
+
+            - english
+
+               - admin_account.php
+
+
+               - more php files
+
+               - extra_definitions
+
+                   - ckeditor.php
+
+                   - more php files
+
+               - images
+
+                   - buttons
+
+                       - button_add_profile.gif
+
+                       - more gif files
+
+               - modules
+
+                   - newsletters
+
+                       - newsletter.php 
+
+                       - more php files
+
+- includes
+
+    - languages
+
+        - english.php
+
+        - english
+
+           - account.php
+
+           - more php files
+
+           - classic
+
+               - header.php
+
+           - extra_definitions
+
+               - cardinal3dsecure.php
+
+               - more php files
+
+               - classic
+
+                   - empty.txt
+               - responsive_classic
+
+                   - product_free_shipping.php
+
+           - html_incldues
+
+               - define_ask_a_question.php
+
+               - more php files
+
+               - classic
+
+                   - define_checkout_success.php
+
+                   - more php files
+
+               - responsive_classic
+
+                   - define_checkout_success.php
+
+                   - more php files
+
+           - images
+
+               - en_CA.gif
+
+               - more gif files

```

Comment:
For most languages there's only 1 gif file in this directory. I'd suggest only mentioning `icon.gif` here. The other english ones are provided as a courtesy. Don't document them.